mappings/net/minecraft/world/gen/ChunkRandom.mapping

```diff
@@ -2,32 +2,77 @@ CLASS net/minecraft/class_2919 net/minecraft/world/gen/ChunkRandom
 	FIELD field_13276 sampleCount I
 	METHOD <init> (J)V
 		ARG 1 seed
-	METHOD method_12659 setSeed (II)J
-		ARG 1 x
-		ARG 2 z
+	METHOD method_12659 setTerrainSeed (II)J
+		COMMENT Seeds the randomizer to generate the surface terrain blocks(such as grass, sand, etc)
+		COMMENT and the bedrock patterns.
+		COMMENT
+		COMMENT <p>Note that the terrain seed is NOT world seed dependant and only gets affected by
+		COMMENT chunk coordinates.</p>
+		ARG 1 chunkX
+		ARG 2 chunkZ
 	METHOD method_12660 consume (I)V
+		COMMENT Skips the provided number of calls to the randomizer.
+		COMMENT
+		COMMENT <p>The skips give the effect of "scrambling" the randomizer but the output is still
+		COMMENT linearly dependant. Note that since multiple calls to a linear congruential generator is
```

Comment:
```suggestion
		COMMENT linearly dependent. Note that since multiple calls to a linear congruential generator is
```
Also, I'm not sure that "linearly dependent" is the correct term to use here. Linear dependence relates to how a vector can be written as the weighted sum of other vectors, how does that apply here?